package.json

```diff
@@ -124,10 +124,11 @@
   },
   "scripts": {
     "gen:parsers": "yarn --cwd packages/kas gen:parsers",
-    "build": "yarn gen:parsers && rollup -c config/build/rollup.config.js",
+    "prebuild": "yarn gen:parsers",
+    "build": "rollup -c config/build/rollup.config.js",
     "build:types": "yarn tsc --build tsconfig-build.json",
-    "build:prodsizecheck": "yarn gen:parsers && rollup -c config/build/rollup.config.js --configEnvironment='production'",
-    "watch": "rollup -c config/build/rollup.config.js --watch",
+    "build:prodsizecheck": "yarn build --configEnvironment='production'",
+    "watch": "yarn build --watch",
```

Comment:
This is just a bit of simplification and deduplication.  `prebuild` is automatically run by script runners any time `build` is requested. 

The other scripts can leverage the `build` script by adding on extra params as needed.